block/src/Pos/Block/BHelpers.hs

```diff
@@ -99,7 +98,6 @@ verifyMainBlock block@UnsafeGenericBlock {..} = do
         verifySscPayload
             (Right (Some _gbHeader))
             (_mbSscPayload _gbBody)
-    dlgVerifyPayload (_gbHeader ^. epochIndexL) (_mbDlgPayload _gbBody)
```

Comment:
Where is it done now?

Reply:
`dlgVerifyPayload` was doing only one check (verifying that psk's epoch index matches current epoch), which was duplicated. The other (now the only) copy is in `dlgVerifyPskHeavy`.